### DISCUSSION TITLE
Hp bar obscuring last cursor position from view when unpausing

### DISCUSSION THREAD

**Discussion by andrey6498** [ppy/osu#18153] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I managed to pause game somewhat before circles appeared and i was unable to unpause afterwards. 

### Screenshots or videos

![osu_2022-05-08_05-10-40](https://user-images.githubusercontent.com/8511369/167279438-3ce57640-e7c7-4641-bd6c-a9e6533a2851.jpg)

### Version

2022.501.0

### Logs

[logs.zip](https://github.com/ppy/osu/files/8646242/logs.zip)

**Comment by andrey6498** [comment]
Referencing #17344 as it's may be related.

**Comment by frenzibyte** [comment]
The logs you attached don't show entering a player session? can you please attach the beatmap, and also try opening up the FPS graph (Ctrl+F11) and attach an image snippet here?

**Reply by andrey6498** [reply]
https://osu.ppy.sh/beatmapsets/883713#osu/1847261

**Reply by andrey6498** [reply]
I guess i know when this possibly happens. There's some timeframe when game treats it as safe/break zone, resulting in not recording last cursor position but it might overlap somewhere there causing softlock(as it usually doesn't require to click in map's break section)

**Reply by peppy** [reply]
Please attach logs again, after reproducing the issue. The logs you provided don't paint a picture of what is going on here.

**Reply by andrey6498** [reply]
I don't think i can reproduce it myself though, since this happened only once so far.

**Reply by andrey6498** [reply]
> I don't think i can reproduce it myself though, since this happened only once so far.

Probably happened right before circles start rendering as they also appear at about -1s

**Reply by andrey6498** [reply]
Oh wait... Now i know the issue.
Hp bar obscures cursor from view

**Comment by andrey6498** [comment]
![osu_2022-05-10_13-21-10](https://user-images.githubusercontent.com/8511369/167609588-a2e77fcf-fab0-4a3b-8f84-e67690919f75.jpg)

**Comment by andrey6498** [comment, marked as answer]
Health bar obscuring last cursor position from view resulting in confusion(you actually still able to unpause).

**Reply by andrey6498** [reply]
@frenzibyte fyi, since it's still may be considered as an UI bug as cursor shouldn't be under hp bar when unpausing.
